src/relayer/Relayer.ts

```diff
@@ -1234,4 +1237,10 @@ export class Relayer {
 
     return msg;
   }
+
+  private getMulticaller(chainId: number) {
```

Comment:
Might want a type annotation for this?

```suggestion
  private getMulticaller(chainId: number): MultiCallerClient {
```